File: yukicoder/369_足し間違い/Main.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int N = sc.nextInt();

        int trueAnswer = 0;

        for (int i = 0; i < N; i++) {
            trueAnswer += sc.nextInt();
        }

        int yukiAnswer = sc.nextInt();

        int diff = trueAnswer - yukiAnswer;

        System.out.println(diff);

    }

}

File: yukicoder/369_足し間違い/Main.cpp
#include <iostream>

using namespace std;

int main() {

    int N;
    cin >> N;

    int trueAnswer = 0;

    for (int i = 0; i < N; i++) {
        int tmp;
        cin >> tmp;

        trueAnswer += tmp;
    }

    int yukiAnswer;
    cin >> yukiAnswer;

    cout << trueAnswer - yukiAnswer;

    return 0;

}
